workflow/rules/qc.smk

```diff
@@ -44,6 +44,7 @@ rule plot_all_samples:
             category="Quality control",
             caption="../report/nanoplot_all_samples_report.rst",
         ),
+    # This parameter is in line with the Snakemake docs 8.20.3 guideline on how to avoid having parameters as output prefixes
```

Comment:
Sorry, I meant to say: Please indicate a sample string, to which index refers.

You are absolutely right, yet I have reason to fear the day, that I have to maintain this piece and I will need hours to pinpoint the change leading to a difficult to understand traceback.

Reply:
will merge regardless